helm/charts/keto/templates/ingress-read.yaml

```diff
@@ -1,42 +1,52 @@
 {{- if .Values.ingress.read.enabled -}}
 {{- $fullName := include "keto.fullname" . -}}
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
I don't think so. Remember that in some versions, you will have both ApiVersions, and only use the newest one once the older versions has been deleted. Reacting on k8s version I think makes us more proactive, as we can react to changes faster

Reply:
It might be, yes, but are you 100% certain that `.Capabilities.KubeVersion.GitVersion` will always be in the format `1.19-0` ?  Cause I'm not 100% sure and in such cases this if won't give meaningful result. 
You are right, we do not provide a way for user to explicitly tell which version to use so we can either : 
- change it to provide some additional flag - then user can impact decision with overrides
- go with apiversion check for newest stable version first -  still does not allow to pick version but ensures stable API first over alpha/beta

Reply:
After some digging: the GitVersion is taken from the git release tag, which uses semVer, so the format is like ` GitVersion:"v1.21.1"`. However the function [semverCompare](https://helm.sh/docs/chart_template_guide/function_list/#semvercompare) is flexible enough for it to work

Reply:
I was reflecting more to forks of K8s as kubernetes providers do not use 1:1 OS k8s ;-)